charon/config.py

```diff
@@ -133,22 +158,20 @@ def get_detach_signature_command(self) -> str:
     def is_aws_cf_enable(self) -> bool:
         return self.__aws_cf_enable
 
```

Comment:
I'm thinking we should add a __is_radas_enabled field here for convenient, which set to "self.__radas_config__ and self.__radas_config__.validate()". So then we can use this with a get method "is_radas_enabled()" for unified way in other place to check if radas is usable.